Nodejs/Product/Nodejs/Snippets/1033/Nodejs/express_get.snippet

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>express_get</Title>
+      <Description>Code Snippet for handling a GET request with Express</Description>
+      <Shortcut>express_get</Shortcut>

```

Comment:
This shortcut isn't very short or easy to type. how about ```expget```,  ```exppost```, ``expput``, and ```expdelete```?